src/components/input-time/input-time.e2e.ts

```diff
@@ -0,0 +1,360 @@
+
+    const inputTime = await page.find("calcite-input-time");
+    const inputEvent = await inputTime.spyOnEvent("calciteInputTimeInput");
+    const changeEvent = await inputTime.spyOnEvent("calciteInputTimeChange");
+
+    await page.keyboard.press("Tab");
+    await page.keyboard.press("1");
+    await page.waitForChanges();
+
+    expect(inputEvent).toHaveReceivedEventTimes(0);
+    expect(changeEvent).toHaveReceivedEventTimes(0);
+
+    await page.keyboard.press("Tab");
+    await page.keyboard.press("2");
+    await page.waitForChanges();
+
+    expect(inputEvent).toHaveReceivedEventTimes(1);
+    expect(changeEvent).toHaveReceivedEventTimes(0);
+
+    await page.keyboard.press("Tab");
+    await page.keyboard.press("3");
+    await page.waitForChanges();
+
+    expect(inputEvent).toHaveReceivedEventTimes(2);
+    expect(changeEvent).toHaveReceivedEventTimes(0);
+
+    await page.keyboard.press("Enter");
+    await page.waitForChanges();
+
+    expect(inputEvent).toHaveReceivedEventTimes(2);
+    expect(changeEvent).toHaveReceivedEventTimes(1);
+
+    await page.keyboard.press("Enter");
+    await page.waitForChanges();
+
+    expect(inputEvent).toHaveReceivedEventTimes(2);
+    expect(changeEvent).toHaveReceivedEventTimes(1);
+
+    await page.keyboard.press("Enter");
+    await page.waitForChanges();
+
+    expect(inputEvent).toHaveReceivedEventTimes(2);
+    expect(changeEvent).toHaveReceivedEventTimes(1);
+  });
+
+  it("Fires change event on blur", async () => {
+    const page = await newE2EPage();
+    await page.setContent(`<calcite-input-time step="1"></calcite-input-time>`);
+
+    const inputTime = await page.find("calcite-input-time");
+    const inputEvent = await inputTime.spyOnEvent("calciteInputTimeInput");
+    const changeEvent = await inputTime.spyOnEvent("calciteInputTimeChange");
+
+    await page.keyboard.press("Tab");
+    await page.keyboard.press("1");
+    await page.waitForChanges();
+
+    expect(inputEvent).toHaveReceivedEventTimes(0);
+    expect(changeEvent).toHaveReceivedEventTimes(0);
+
+    await page.keyboard.press("Tab");
+    await page.keyboard.press("2");
+    await page.waitForChanges();
+
+    expect(inputEvent).toHaveReceivedEventTimes(1);
+    expect(changeEvent).toHaveReceivedEventTimes(0);
+
+    await page.keyboard.press("Tab");
+    await page.keyboard.press("3");
+    await page.waitForChanges();
+
+    expect(inputEvent).toHaveReceivedEventTimes(2);
+    expect(changeEvent).toHaveReceivedEventTimes(0);
+
+    await page.keyboard.press("Tab");
+    await page.keyboard.press("Tab");
+    await page.waitForChanges();
+
+    expect(inputEvent).toHaveReceivedEventTimes(2);
+    expect(changeEvent).toHaveReceivedEventTimes(1);
+
+    await page.keyboard.down("Shift");
+    await page.keyboard.press("Tab");
+    await page.keyboard.up("Shift");
+    await page.keyboard.press("Tab");
+    await page.waitForChanges();
+
```

Comment:
you can remove all the `expect`s before this one to improve the speed of the tests without losing test integrity. If you want to better describe what is happening, add some inline comments.